src/App.js

```diff
@@ -1,32 +1,28 @@
 import React, { Fragment } from "react";
 import CssBaseline from "@material-ui/core/CssBaseline";
 
-import { Grid } from "@material-ui/core";
-import HomePage from "./components/home-page.jsx";
-import Footer from "./components/common/footer.jsx";
 import NavDrawer from "./components/common/nav-drawer";
+import Footer from "./components/common/footer.jsx";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import GoogleMapsPage from "./components/google-map/google-map-page.jsx";
 
+import HomePage from "./components/home-page.jsx";
+import FormPage from "./components/form/form-page.jsx";
+
 function App() {
   return (
     <Router>
       <CssBaseline />
       <NavDrawer
         pageContent={
           <Fragment>
-            <Grid
-              container
-              direction="column"
-              justifyContent="center"
-              alignItems="center"
-              style={{ minHeight: "100vh" }}
-            >
-              <Switch>
-                <Route exact path="/" component={HomePage} />
-                <Route path="/google-map-page" component={GoogleMapsPage} />
-              </Switch>
-            </Grid>
+
+            <Switch>
+              <Route exact path="/" component={HomePage} />
+              <Route path="/google-map-page" component={GoogleMapsPage} />
+              <Route path="/form-page" component={FormPage} />
+            </Switch>
+
```

Comment:
Any reason of moving the initial grid container? It is the base for each page.

Reply:
I was thinking that if a single page does not require a grid or even if it does but with different styles so that it can be changed in those individual pages. There is no restriction to think about the base grid.